packages/roosterjs-editor-core/lib/corePlugins/ImageSelection.ts

```diff
@@ -43,7 +43,7 @@ export default class ImageSelection implements EditorPlugin {
                     if (
                         safeInstanceOf(target, 'HTMLImageElement') &&
                         target.isContentEditable &&
-                        event.rawEvent.button === mouseLeftButton
+                        event.rawEvent.button != mouseMiddleButton
```

Comment:
There is a copy of this file under content model folder: https://github.com/microsoft/roosterjs/blob/master/packages-content-model/roosterjs-content-model-adapter/lib/corePlugins/ImageSelection.ts

Would you please do the same change to the copied file? Thanks.

Reply:
done